dags.yaml

```diff
@@ -117,6 +117,21 @@ bqetl_mozilla_vpn:
     retries: 2
     retry_delay: 30m
 
+bqetl_mozilla_vpn_site_metrics:
+  schedule_interval: 0 23 * * *
+  description: |
+    Daily extracts from the Google Analytics tables for Mozilla VPN as well as
+    derived tables based on that data.
+
+    Depends on Google Analytics exports, which have highly variable timing, so
+    is scheduled to run very late in the UTC day.
```

Comment:
over the 155 days since the GA exports were enabled, these are the UTC hours when the export for the full day became available:

| export_hour | days |
|-------------|------|
|    23:00:00 |    1 |
|    21:00:00 |   32 |
|    18:00:00 |   13 |
|    17:00:00 |   36 |
|    16:00:00 |   40 |
|    15:00:00 |   28 |
|    14:00:00 |    5 |